tests/build_contract.rs

```diff
@@ -16,7 +16,8 @@ fn setup_test_environment() -> Result<tempfile::TempDir, Error> {
 }
 
 #[test]
-fn test_contract_build() -> Result<(), Error> {
+#[cfg_attr(not(feature = "e2e_contract"), ignore)]
```

Comment:
Instead of `#[cfg_attr(not(feature = "e2e_contract"), ignore)]` on top of every test, one module `cfg` : `#![cfg(feature = "e2e_contract")]` suffices.